client/jetpack-connect/jetpack-new-site/index.jsx

```diff
@@ -92,9 +95,13 @@ class JetpackNewSite extends Component {
 							<div className="jetpack-new-site__card-description">
 								{ this.props.translate( 'We’ll be using the Jetpack plugin to connect your site to WordPress.com.' ) }
 							</div>
-							<SiteURLInput ref="siteUrlInputRef"
-								onTosClick={ this.handleOnClickTos }
-								onClick={ this.onURLEnter } />
+							<SiteURLInput
+								ref="siteUrlInputRef"
```

Comment:
Shouldn't be, but doesn't hurt. I'd like to deploy and handle extra clean up later 😅